docker/Dockerfile.dev

```diff
@@ -17,7 +17,7 @@ COPY package.json package-lock.json /usr/local/hypha/
 RUN sudo chown -R circleci:circleci /usr/local
 
 # Install node dependencies.
-RUN npm install  --quiet --global gulp-cli
+RUN npm install  --quiet --global 
```

Comment:
This line can be removed since it was only for installing `gulp-cli`

Reply:
sure will update